src/components/WorkPoolQueueStatusIcon.vue

```diff
@@ -17,12 +17,15 @@
   import { Icon } from '@prefecthq/prefect-design'
   import { computed } from 'vue'
   import StatusIcon from '@/components/StatusIcon.vue'
+  import { useWorkPool } from '@/compositions/useWorkPool'
   import { WorkPoolQueue } from '@/models'
 
   const props = defineProps<{
     workPoolQueue: WorkPoolQueue,
   }>()
 
+  const { workPool } = useWorkPool(props.workPoolQueue.workPoolName)
```

Comment:
Assuming this composition accepts a getter you'll want to use one to not break reactivity if the prop changes. 
```suggestion
  const { workPool } = useWorkPool(() => props.workPoolQueue.workPoolName)
```